devicetypes/smartthings/aeotec-doorbell-6.src/aeotec-doorbell-6.groovy

```diff
@@ -0,0 +1,238 @@
+        try {
+            String childDni = "${device.deviceNetworkId}:$endpoint"
+            def componentLabel = device.displayName[0..-2] + "$endpoint"
+
+            addChildDevice("Aeotec Child Siren", childDni, device.getHub().getId(), [
+                    completedSetup: true,
+                    label         : componentLabel,
+                    isComponent   : true,
+                    componentName : "Aeotec Doorbell 6 $endpoint",
+                    componentLabel: "Aeotec Doorbell 6 $endpoint"
+            ])
+        } catch (Exception e) {
+            log.debug "Excep: ${e} "
+        }
+    }
+}
+
+private encap(cmd, endpoint = null) {
+    if (endpoint) {
+        zwave.multiChannelV3.multiChannelCmdEncap(destinationEndPoint: endpoint).encapsulate(cmd)
+    } else {
+        cmd
+    }
+}
+
+def zwaveEvent(physicalgraph.zwave.Command cmd, endpoint = null) {
+    log.warn "Unhandled ${cmd}" + (endpoint ? " from endpoint $endpoint" : "")
+}
+
+def sendCommand(deviceDni, commands) {
+    def result = commands.collect {
+        if (it instanceof String) {
+            it
+        } else {
+            encap(it, channelNumber(deviceDni))
+        }
+    }
+    sendHubCommand(result, 100)
+}
+
+def channelNumber(String dni) {
+    dni?.split(":")[-1] as Integer
+}
+
+
+def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cmd) {
+    if (cmd.notificationStatus == 0xFF) {
+        switch (cmd.event) {
+            case 0x09: //TAMPER
+                sendEvent([name: "tamper", value: "detected"])
+                break
+            case 0x01: //ON
+                if (state.soundTable[1]) {
+                    sendEvent([name: "alarm", value: "both"])
+                } else {
+                    resetActiveSounds()
+                    String childDni = "${device.deviceNetworkId}:$state.lastTriggeredSound"
+                    def child = childDevices.find { it.deviceNetworkId == childDni }
+                    child?.sendEvent([name: "alarm", value: "both"])
+                }
+                break
+            case 0x00: //OFF
+                if (state.soundTable[1]) {
+                    sendEvent([name: "alarm", value: "off"])
+                    sendEvent([name: "tamper", value: "clear"])
+                } else {
+                    childDevices.each {
+                        String childDni = "${device.deviceNetworkId}:$state.lastTriggeredSound"
+                        def child = childDevices.find { it.deviceNetworkId == childDni }
+                        child?.sendEvent([name: "alarm", value: "off"])
+                    }
+                }
+                break
+        }
+    }
+}
+
+def setOnChild(deviceDni, value) {
+    state.lastTriggeredSound = deviceDni?.split(":")[-1] as Integer
+    setActiveSound(state.lastTriggeredSound)
+    resetActiveSounds()
+    sendHubCommand sendCommand(deviceDni, zwave.basicV1.basicSet(value: value))
```

Comment:
`sendCommand` already calls `sendHubCommand`. I would remove it here instead of there.